tests/lib/Sets/Ibexa50/Fixture/some_class.php.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Ibexa\Rector\Tests\Sets\Ibexa50\Fixture;
+
+use Ibexa\Bundle\Core\ApiLoader\RepositoryConfigurationProvider;
+
+class FooBar {
+    public function foo(): RepositoryConfigurationProvider
+    {
+        return $this->bar(RepositoryConfigurationProvider::class);
+    }
+
+    public function bar(string $class): RepositoryConfigurationProvider
+    {
+        return new $class();
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Ibexa\Rector\Tests\Sets\Ibexa50\Fixture;
+
+use Ibexa\Bundle\Core\ApiLoader\RepositoryConfigurationProvider;
+
+class FooBar {
+    public function foo(): \Ibexa\Contracts\Core\Container\ApiLoader\RepositoryConfigurationProviderInterface
+    {
+        return $this->bar(\Ibexa\Contracts\Core\Container\ApiLoader\RepositoryConfigurationProviderInterface::class);
```

Comment:
Note: Rector per design expects that a Developer runs CS fixer afterwards. For the automated PR I'll run CS fixer with extra FQCN import rule, which will both remove unused `use` and simplify FQCN.